Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
 
 # Install prerequisites
 RUN set -x && \
-  apt-get update && apt-get -y upgrade && \
```

Comment:
Yeah, I wanted to call this out because I could see you wanting to keep it in. Feel free to drop that part.